rust/opendp/src/meas/stability.rs

```diff
@@ -103,14 +103,15 @@ mod test_stability {
     use super::*;
 
     #[test]
-    fn test_base_stability() {
+    fn test_base_stability() -> Fallible<()> {
```

Comment:
Is there any issue with having this return `Fallible`? If the ? causes an early return, won't the rest of the test code be skipped? Or does the test harness treat an `Err` return value as a failure?

Reply:
The test harness treats the Err return value as a failure. See more here: https://doc.rust-lang.org/edition-guide/rust-2018/error-handling-and-panics/question-mark-in-main-and-tests.html

Reply:
OK great, good to know.